Extensions/magiccards_popups.js

```diff
@@ -0,0 +1,93 @@
+//* TITLE magiccards.info Popups **//
+//* VERSION 0.1.4 **//
+//* DESCRIPTION    Creates visual popups for links to cards on https://magiccards.info/. **//
+//* DEVELOPER thebombzen **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.magiccards_popups = new Object({
+
+	running: false,
+
+	run: function() {
+		this.running = true;
+		if (XKit.interface.is_tumblr_page() === true) {
+			XKit.extensions.magiccards_popups.addPopupsIfNecessary();
+			XKit.tools.init_css("magiccards_popups");
+			XKit.post_listener.add("magiccards_popups", XKit.extensions.magiccards_popups.addPopupsIfNecessary);
+		}
+	},
+
+	destroy: function() {
+		this.running = false;
+		XKit.extensions.magiccards_popups.removePopups();
+		XKit.tools.remove_css("magiccards_popups");
+		XKit.post_listener.remove("magiccards_popups");
+	},
+
+	magiccards_regex: new RegExp("^https?://magiccards.info/([^/]+)/([^/]+)/([^.]+).html"),
+
+	addPopupsIfNecessary: function() {
+		$("a").each(function() {
```

Comment:
Okay, I wrote a multi-line comment on how I thought this line would be far too slow to be run on any new post but it turns out it runs in less than 10ms on average so 👍